### PR TITLE
docs(contributing): fix a broken link

### PR DESCRIPTION
**Pull request by NouiliKh**
In the contributing.md file, there is a broken link that needs to be changed:

image

I will just change which link jina-101 is referring to: from "./101/README.html" to  "https://github.com/jina-ai/jina/tree/master/docs/chapters/101"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ In this guide we're going to go through the steps for each kind of contribution,
 <a name="-before-you-start"></a>
 ## 🏁 Before you Start
 
-Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](./101/README.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
+Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://github.com/jina-ai/jina/tree/master/docs/chapters/101), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
```
Comment (nan-wang) [inline CONTRIBUTING.md:30]:
```suggestion
Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://docs.jina.ai/chapters/101/index.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
It is better to refer to the URL at docs.jina.ai because the file in the repository might be subject to change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (NouiliKh) [reply CONTRIBUTING.md:30]: @nan-wang It makes sense. 
Taking this into consideration, I think we should change: https://github.com/jina-ai/examples/blob/master/README.md as well.
image

**Comment by NouiliKh** [comment]
@nan-wang It makes sense. 
Taking this into consideration, I think we should change: https://github.com/jina-ai/examples/blob/master/README.md as well.
image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ In this guide we're going to go through the steps for each kind of contribution,
 <a name="-before-you-start"></a>
 ## 🏁 Before you Start
 
-Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](./101/README.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
+Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://docs.jina.ai/chapters/101/index.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
```
Comment (hanxiao) [inline CONTRIBUTING.md:30]:
```suggestion
Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://101.jina.ai), and [example tutorials](https://learn.jina.ai) so you have a good understanding of what Jina is and how it works.
```

Comment (hanxiao) [reply CONTRIBUTING.md:30]: @nan-wang we have those shortcuts

**Pull request merged**